rule/get-return.go

```diff
@@ -0,0 +1,71 @@
+package rule
+
+import (
+	"fmt"
+	"go/ast"
+	"strings"
+
+	"github.com/mgechev/revive/lint"
+)
+
+// GetReturnRule lints given else constructs.
+type GetReturnRule struct{}
+
+// Apply applies the rule to given file.
+func (r *GetReturnRule) Apply(file *lint.File, arguments lint.Arguments) []lint.Failure {
+	var failures []lint.Failure
+
+	onFailure := func(failure lint.Failure) {
+		failures = append(failures, failure)
+	}
+
+	w := lintReturnRule{onFailure}
+	ast.Walk(w, file.AST)
+	return failures
+}
+
+// Name returns the rule name.
+func (r *GetReturnRule) Name() string {
+	return "get-return"
+}
+
+type lintReturnRule struct {
+	onFailure func(lint.Failure)
+}
+
+func isGetter(name string) bool {
+	if strings.HasPrefix(strings.ToUpper(name), "GET") {
+		if len(name) > 3 {
+			c := name[3]
+			return (c >= 'A' && c <= 'Z')
```

Comment:
Why ignoring special chars?

Reply:
`isGetter` is not perfect in determining if a function is a getter or not. Currently it just checks for patterns like `getSomething` or `GetSomething`. Do you think it needs to also check for special characters after `get`?   We can simply check that the fourth character is not lowercase?

```go
if strings.HasPrefix(strings.ToUpper(name), "GET") {
	if len(name) > 3 {
		c := name[3]
		return (c >= 'a' && c <= 'z')
```

Reply:
Oh what I meant was if it'll work with:

```
get_stuff
```

Reply:
Ok, I see what you mean. I will modify the function to cope with such cases

Reply:
I've searched on Docker, Kubernetes, and Go code bases and I did not found getters using _ or other character after the `get` prefix
I've modified to `isGetter` to check that the fourth character is not in [a-z].